Update interfaces/wake-lock.idl

autofoolip opened the pull request:
This PR was automatically created by a bot.

Before merging, please check that any tests that depend on the updated IDL files still work.

If additional changes are needed, please manually create another PR based on this one.

See the [README](https://github.com/web-platform-tests/wpt/blob/master/interfaces/README.md) for how the IDL files in this directory are used.

<hr>

Source: https://github.com/tidoust/reffy-reports/blob/52f31b9/whatwg/idl/wake-lock.idl
Build: https://travis-ci.org/tidoust/reffy-reports/builds/563479802

lukebjerring commented:
This is incorrect; an instance of https://github.com/tidoust/reffy-reports/issues/34 caused by https://github.com/w3c/wake-lock/commit/5c34f9a299ad4625fe32398152bf85ab8cdf1c24

autofoolip committed:
Update interfaces/wake-lock.idl

Source: https://github.com/tidoust/reffy-reports/blob/52f31b9/whatwg/idl/wake-lock.idl
Build: https://travis-ci.org/tidoust/reffy-reports/builds/563479802

foolip commented:
It has recovered, changes make sense now.
